### PR TITLE
Add sanitize levels to sanitization routines at large

### PR DESCRIPTION
**Pull request by hecon5**
This implements generic sanitziation levels, and allows you to totally turn off all sanitization.

WIP: need to add detection for 'old' sanitize level and new setting conversion.

### CODE REVIEW

**Comment by hecon5** [comment]
Fixes  #239

**Comment by hecon5** [comment]
@joyfullservice this is ready for review; let me know your thoughts!

File: Version Control.accda.src/forms/frmVCSOptions.bas [hunk 1]
```diff
@@ -3696,14 +3709,25 @@ Private Sub Form_Load()
...
-        For intColorSanitizeLevel = 0 To (eSanitizeColors.[_Last] - 1)
-            .AddItem intColorSanitizeLevel & "," & Options.GetColorSanitizeLevelName(intColorSanitizeLevel)
-        Next intColorSanitizeLevel
+        For intSanitizeLevel = 0 To (eSanitizeLevel.[_Last] - 1)
+            .AddItem intSanitizeLevel & "," & Options.GetSanitizeLevelName(intSanitizeLevel)
+        Next intSanitizeLevel
     End With
 
+    ' Set focus to the cancel button; this avoids weird highlighting of options boxes.
+    Me.cmdCancel.SetFocus
```
Comment (joyfullservice) [inline Version Control.accda.src/forms/frmVCSOptions.bas:3729]: Not really seeing the issue here... I fixed a couple things with the tab order, but it seems to be working as intended.

Comment (hecon5) [reply Version Control.accda.src/forms/frmVCSOptions.bas]: For some reason, for a while the new Sanitize Level box was highlighted when clicking on that tab; it was annoying so I did that and the issue went away. I can remove it after some quick testing.

**Comment by joyfullservice** [comment]
This looks good to me.  👍  Are you ready for me to merge this to `dev`?

**Comment by hecon5** [comment]
I have a few touch ups, updating the options wiki right now; be about 15 minutes!

**Comment by hecon5** [comment]
ok, give that a run, let me know if it looks good!

**Comment by joyfullservice** [comment]
Great work, @hecon5! I think this will be a meaningful long-term enhancement to the project. Thanks for your contributions!  👏

**Pull request merged**